package.json

```diff
@@ -34,7 +34,8 @@
     "underscore": ">= 1.3.1",
     "uuid": "^3.1.0",
     "xmldom": ">= 0.1.x",
-    "xpath.js": "~1.1.0"
+    "xpath.js": "~1.1.0",
+    "axios": "^0.21.1"
```

Comment:
Be sure to remove `request` as well.

Reply:
I kept it to fix the last failing unit test, request is making one less mock nw call than axios. Will change soon after I fix it.

Reply:
Can you review now and approve?